django_freeradius/base/admin.py

```diff
@@ -206,7 +206,26 @@ def get_readonly_fields(self, request, obj=None):
 
 
 class AbstractRadiusProfileAdmin(TimeStampedEditableAdmin):
-    pass
+    list_display = ['name', 'get_daily_session_limit', 'get_daily_bandwidth_limit', 'max_all_time_limit',
```

Comment:
@R9295 I think it should be `get_max_all_time_limit` and not `max_all_time_limit`. Writing tests would have been useful here.